src/push-notifications/fcm/PushNotificationsFcmApi.ts

```diff
@@ -0,0 +1,59 @@
+import type { FcmDeviceInfo } from './models'
+
+import { OutboundMessageContext, AgentContext, ConnectionService, injectable, MessageSender } from '@credo-ts/core'
+
+import { PushNotificationsFcmService } from './services/PushNotificationsFcmService'
+import {
+  PushNotificationsFcmDeviceInfoHandler,
+  PushNotificationsFcmProblemReportHandler,
+  PushNotificationsFcmSetDeviceInfoHandler,
+} from './handlers'
+
+@injectable()
+export class PushNotificationsFcmApi {
```

Comment:
How does this compare to https://github.com/openwallet-foundation/credo-ts-ext/tree/main/packages/push-notifications/src/fcm?

Reply:
`credo-ts-ext` doesn't have a Repository to store record so we added other files with [https://github.com/credebl/mediator-agent/tree/refactor/mediator/src/push-notifications/fcm/repository](https://github.com/credebl/mediator-agent/tree/refactor/mediator/src/push-notifications/fcm/repository). We will add this repository into credo-ext and later remove from mediator